include/deal.II/grid/manifold.h

```diff
@@ -32,6 +33,38 @@ DEAL_II_NAMESPACE_OPEN
 // forward declaration
 template <int, typename> class Table;
 
+namespace internal
+{
+  template<int dim>
+  inline
+  constexpr int n_default_points_per_cell()
+  {
+    return std::numeric_limits<int>::max();
+  }
+
+  template <>
+  inline
+  constexpr int n_default_points_per_cell<1>()
+  {
+    return 2;
+  }
+
+  template <>
+  inline
+  constexpr int n_default_points_per_cell<2>()
+  {
+    return 8;
+  }
+
+  template <>
+  inline
+  constexpr int n_default_points_per_cell<3>()
+  {
+    return GeometryInfo<3>::vertices_per_cell + GeometryInfo<3>::lines_per_cell
+           + GeometryInfo<3>::faces_per_cell;
```

Comment:
Why not simply returning `3^dim-1`?

Reply:
I would prefer to write it out this way so that it is clear what the number signifies.